.changeset/lucky-trainers-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@vercel/node": patch
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** please ‏make sure there are lines for `cli` and `edge` packages as well

Reply:
added although not sure why; changeset added what it detects.

Reply:
Sorry, I don't see the updated file. Was this updated to include `cli` and `edge`?